src/server.ts

```diff
@@ -14,6 +16,8 @@ const app = express()
 
 app.get('/profile', findOrCreateProfile)
 
+app.use(express.static('frontend/build'))
```

Comment:
Issue: this won't work on nested routes, like https://shipment-tracker-dev.herokuapp.com/groups for example. I think we need to tell Express to render every route statically?

```ts
// I'm not familiar with the `.static` method but this might work?
app.use('*', express.static('frontend/build'))

// Alternate solution:
app.get('*', (req, res) => {
  res.sendFile(path.join(__dirname, 'frontend/build')
}
```

Reply:
I'll give it a shot

Reply:
Done